.changes/unreleased/tinymce-TINY-10436-2024-02-27.yaml

```diff
@@ -0,0 +1,7 @@
+project: tinymce
+kind: Removed
+body: Removed `force_hex_color` option, with the default now being all colors are forced
```

Comment:
We never released this feature so this changelog item should just mention what the new behavior is vs the previous release. The `force_hex_color` option was never something that the users saw.

Reply:
we did, it was added late in the 6.x cycle once a customer complained about the regression
https://www.tiny.cloud/docs/tinymce/latest/content-formatting/#force_hex_color